private_gpt/settings/settings.py

```diff
@@ -217,6 +217,31 @@ class QdrantSettings(BaseModel):
     )
 
 
+class SQLDatabaseSettings(BaseModel):
+    enabled: bool = Field(
+        False,
+        description="Flag to enable SQL Query mode. Disabled by default",
+    )
+    dialect: Literal["mysql"]
+    driver: Literal["pymysql"]
```

Comment:
This is going against the documentation - Pydantic will fail to validate any other values, and consequently, one will not be able to change the dialect and driver.